hack/release/make-release-tag.sh

```diff
@@ -47,10 +47,12 @@ for example in examples/contour/03-envoy.yaml examples/deployment/03-envoy-deplo
 done
 
 # Update the certgen job name to the new version.
+# Dot in the version is replaced with a dash to make it a valid pod name.
 for example in examples/contour/02-job-certgen.yaml ; do
     run::sed \
-        "-es|contour-certgen-main|contour-certgen-$NEWVERS|" \
-        "-es|contour-certgen-$OLDVERS|contour-certgen-$NEWVERS|" \
+        "-es|contour-certgen-main|contour-certgen-${NEWVERS//./-}|" \
+        "-es|contour-certgen-$OLDVERS|contour-certgen-${NEWVERS//./-}|" \
+        "-es|contour-certgen-${OLDVERS//./-}|contour-certgen-${NEWVERS//./-}|" \
```

Comment:
I'm not completely sure if both these are needed? We can have old format with dots when releasing minor release from "old" branch, new version with dash in the future release branches?

Reply:
maybe be nice to do for older release branches, 1.25 supports k8s 1.27 so this warning would apply to that minor release and future patches